Gateway/Response/PaymentDetailsHandler.php

```diff
@@ -37,10 +51,8 @@ public function handle(array $handlingSubject, array $response)
             $payment->setAdditionalInformation('barcode', $barcode);
         }
 
-        if ($response['charge']->source['type'] === 'paynow') {
-            $qrCodeImage = $this->downloadPaymentFile($response['charge']->source['scannable_code']['image']['download_uri']);
-            $payment->setAdditionalInformation('qr_code_encoded', base64_encode($qrCodeImage));
+        if ($this->_helper->isPayableByImageCode($response['charge']->source['type'])) {
```

Comment:
Just idea, I might be wrong but I'm not sure if the `$payment` here is the actual Omise `Model\Config\*` (payment method class), but if so, we probably can implement something like:
```php
<?php
namespace Omise\Payment\Model\Config;

use Omise\Payment\Model\Config\Config;

class Promptpay extends Config
{
    /**
     * @var string
     */
    const CODE = 'omise_offline_promptpay';

    const HAS_SCANNABLE_CODE = true;
}
```

```php
if ($payment::HAS_SCANNABLE_CODE) {
    // ...
}
```

But yeah, I guess I'm wrong... I forgot what is inside `$payment = $payment->getPayment();`